modules/account-lib/src/coin/algo/keyRegistrationBuilder.ts

```diff
@@ -141,6 +157,9 @@ export class KeyRegistrationBuilder extends TransactionBuilder {
       this._voteLast,
       this._voteKeyDilution,
       this.suggestedParams,
+      undefined,
```

Comment:
Add comments with the names of the parameters skipped to help reader understand what's missing